pkg/services/control/types.proto

```diff
@@ -23,6 +23,9 @@ enum NetmapStatus {
 
     // Node is offline.
     OFFLINE = 2;
+
+    // Node is maintained by the owner.
+    MAINTENANCE = 3;
```

Comment:
Do we have a PR in `neofs-api`?

Reply:
`MAINTENANCE` status is not reflected in Netmap contract, so we don't have it in API (at least for now).